modules/running-your-dynamic-plugin.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin"]
+= Running your dynamic plug-in
+You can run the plugin using a local development environment.
+
+
+. Run Bridge locally and instruct it to proxy requests directly to your local plugin asset server (web server hosting the plugin's generated assets): `./bin/bridge -plugins console-demo-plugin=http://localhost:9001/`
```

Comment:
There is more to it than this since you need to clone the openshift/console repo and build the `bridge` binary.

Let me see if I can't come up with some steps that avoid needing to do that.